features/FEATURE_LWIP/lwip-interface/lwip-sys/lwip_random.h

```diff
@@ -0,0 +1,47 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2017 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef LWIP_HDR_RANDOM_H
+#define LWIP_HDR_RANDOM_H
+
+#include "lwip/opt.h"
+#include "lwip/debug.h"
+#include "lwip/def.h"
+
+#if FEATURE_COMMON_PAL
+#include "ns_types.h"
+#include "randLIB.h"
+#endif
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#if FEATURE_COMMON_PAL
+void lwip_seed_random(void);
+void lwip_add_ramdom_seed(uint64_t seed);
+#else
+#define lwip_seed_random()
```

Comment:
why are these defined? I would rather provide an empty implementation ?

will it work fro the random_seed ?